### DISCUSSION TITLE
您好：请教一个问题：我根据官网文档进行学习，可以快速搭建出来2D,但是用官网的3D加载案例一直加载不出来，没有报错

### DISCUSSION THREAD

**Discussion by nyhxiaoning** [galacean/discussion#30] [Engine]
**Describe the bug**
您好我照着官方网站进行了安装，第一步绘制立方体正常展示，找着demo（没有任何删改）进行第二步加载3D和加载脚步，都无法
成功加载，但是一直无法正常显示，控制台没有报错。
想问一下：加载3D和脚步是需要其他的特殊内容吗？
**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**
![image](https://user-images.githubusercontent.com/34833782/200173980-7a339095-796c-4696-b3aa-9a53b58e9d84.png)

**Additional context**

**Comment by cptbtptpbcptdtptp** [comment]
感谢反馈，如果方便可以上传一下你的工程文件吗。

**Comment by nyhxiaoning** [comment]
@cptbtptpbcptdtptp 麻烦看一下，使用了vue3

**Comment by nyhxiaoning** [comment]
[web-game-oasis-engine-master.zip](https://github.com/ant-galaxy/oasis-engine/files/9947666/web-game-oasis-engine-master.zip)

**Comment by eyworldwide** [comment]
@nyhxiaoning 这个 issue 我转移到了 discussion 讨论
